src/pybind/ceph_volume_client.py

```diff
@@ -754,11 +755,12 @@ def rmtree(root_path):
             dir_handle = self.fs.opendir(root_path)
             d = self.fs.readdir(dir_handle)
             while d:
-                if d.d_name not in [".", ".."]:
+                d_name = d.d_name.decode()
```

Comment:
and here, d.d_name.decode(encoding='utf-8')

Reply:
Done.